doc/ledger3.texi

```diff
@@ -4174,6 +4176,29 @@ This example causes 10% of the matching account's total to be deferred
 to the @samp{Savings} account---as a balanced virtual posting, which
 may be excluded from reports by using @option{--real}.
 
+@subsubsection Credit Card Cashback
+
+Credit cards sometimes provide a cashback percentage of purchases. This
+can be setup with the following:
+
+@smallexample @c input:C371854
```

Comment:
The input identifiers should be unique per example to ensure [`test/DocTests`](https://github.com/ledger/ledger/blob/master/test/DocTests.py) work properly; for details see [`doc/ledger3.texi:37ff`](https://github.com/ledger/ledger/blob/master/doc/ledger3.texi#L37:L42).

```suggestion
@smallexample @c input:7D1589F
```